compio-driver/src/sys/unix_op.rs

```diff
@@ -20,6 +20,13 @@ impl OpenFile {
     }
 }
 
+/// [`libc::stat`]
+#[cfg(not(gnulinux))]
+pub use libc::stat as Stat;
+/// [`libc::stat64`]
+#[cfg(gnulinux)]
+pub use libc::stat64 as Stat;
```

Comment:
Should we make this consistent with next line, i.e., `type Stat = ...`?

Reply:
`use` should be preferred if possible, I think. The below one is not exposed to public though.